config/routes.rb

```diff
@@ -1,14 +1,16 @@
 Rails.application.routes.draw do
-  devise_for :users, controllers: { sessions: 'sessions',
-                                    registrations: 'registrations' }
+  devise_for :users, only: []
+  # devise_for :users, controllers: { sessions: 'sessions',
```

Comment:
If you're creating all user devise_for routes here, could you remove the "only: []"? Asking more out of curiosity...